src/assets/translations/en/main.json

```
@@ -2983,6 +2983,18 @@
       "claimAvailable": "Claim Available",
       "unknown": "Unknown"
     },
+    "swap": {
+      "processing": "Your swap of %{sellAmountAndSymbol} to %{buyAmountAndSymbol} is still being processed.",
```

`still` terminology feels odd to read at runtime IMO, I would get that if we were waiting forever for a swap and were reminding users that fundus are safu, but seeing "still" immediately feels off 

```suggestion
      "processing": "Your swap of %{sellAmountAndSymbol} to %{buyAmountAndSymbol} is being processed.",
```

---

good sniff. tighter copy FTW. 🙌

---

Updated in [c30c8e9](https://github.com/shapeshift/web/pull/9738/commits/c30c8e93ba075d595b1d8f868461c4e171aaaa02)